packages/renderer/src/lib/mcp/setup/InputArgument.spec.ts

```diff
@@ -0,0 +1,88 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025 Red Hat, Inc.
```

Reply:
Good catch 👍

Reply:
I hope we could find an eslint rule (not a slow one and a working one to validate this (and auto-fix))

Reply:
I think we had one that we dropped after the switch to eslint 9 but AFAIK it was not checking date, only mainly that we had a copyright header in the file